File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.*;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int max = -1000000;// -1,000,000
        int min = 1000000;
        for(int i = 0; i < n; i++) {
            int a = sc.nextInt();
            if(a > max) max = a;
            if(a < min) min = a;
        }
        System.out.println(min + " " + max);
    }
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.cc
#include<stdio.h>

int main() {
  int n, min = 1000000, max = -1000000, a;
  scanf("%d", &n);
  for (int i = 0; i < n; i++) {
    scanf("%d", &a);
    if (min > a) min = a;
    if (max < a) max = a;
  }
  printf("%d %d", min, max);
}